### PR TITLE
CI Update - mdoc 5.7.3

### PR DESCRIPTION
**Pull request by dend**

### CODE REVIEW

File: xml/Microsoft.VisualBasic/Interaction.xml [hunk 1]
```diff
@@ -1270,14 +1270,14 @@ drawObj.SaveAs("C:\Drawings\sample.drw")
     <Member MemberName="Shell">
       <MemberSignature Language="C#" Value="public static int Shell (string Pathname, Microsoft.VisualBasic.AppWinStyle Style = Microsoft.VisualBasic.AppWinStyle.MinimizedFocus, bool Wait = false, int Timeout = -1);" FrameworkAlternate="netframework-1.1" />
       <MemberSignature Language="ILAsm" Value=".method public static int32 Shell(string Pathname, valuetype Microsoft.VisualBasic.AppWinStyle Style, bool Wait, int32 Timeout) cil managed" FrameworkAlternate="netframework-1.1" />
-      <MemberSignature Language="DocId" Value="M:Microsoft.VisualBasic.Interaction.Shell(System.String,Microsoft.VisualBasic.AppWinStyle,System.Boolean,System.Int32)" FrameworkAlternate="netframework-2.0" />
+      <MemberSignature Language="DocId" Value="M:Microsoft.VisualBasic.Interaction.Shell(System.String,Microsoft.VisualBasic.AppWinStyle,System.Boolean,System.Int32)" FrameworkAlternate="netframework-1.1;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2" />
```
Comment (mairaw) [inline xml/Microsoft.VisualBasic/Interaction.xml]: @joelmartinez the signature is the same for all framework versions. not sure why the 2.0 gets separated. can't remember if we already have an issue tracking this.

File: xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
     </Attribute>
   </Attributes>
   <Parameters>
-    <Parameter Name="" Type="TArg2" Index="0" FrameworkAlternate="netframework-3.5;netframework-4.0" />
+    <Parameter Name="" Type="TArg2" Index="1" FrameworkAlternate="netframework-3.5;netframework-4.0" />
```
Comment (mairaw) [inline xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml]: this change lloks wrong. line 32 already has this.

**Comment by mairaw** [comment]
There are still duplicate members as well. I think this bug won't allow us to index the content for API browser:
https://review.docs.microsoft.com/en-us/dotnet/api/system.servicemodel.clientbase-1.channelbase-1.system-servicemodel-channels-irequestchannel-remoteaddress?view=netframework-4.7.2&branch=pr-en-us-463#System_ServiceModel_ClientBase_1_ChannelBase_1_System_ServiceModel_Channels_IRequestChannel_RemoteAddress

File: xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml (hunk 2 above)
Comment (joelmartinez) [reply xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml]: Hmm, you're right, this is not correct ... looks like in this case, the empty parameter name is making the indexing algorithm go wonky :( 

Seems like it's a slightly different issue than https://github.com/mono/api-doc-tools/issues/291, though unfortunately the [original issue there](https://github.com/dotnet/dotnet-api-docs/blob/0012e5ded56afa6e8fd1f359bc62098b949c91b0/xml/System.IO.Compression/DeflateStream.xml#L397) still remains. So I'm going to re-open that bug

**Comment by joelmartinez** [comment]
@mairaw something is weird here, because if you look at the ecmaxml source, there are only three `<Member` nodes for `RemoteAddress`

- [`.RemoteAddress`](https://github.com/dotnet/dotnet-api-docs/blob/49f51e85e33a589b8aead775f1ac0c47ba2dd27c/xml/System.ServiceModel/ClientBase%601+ChannelBase%601.xml#L480)
- [`.IOutputChannel.RemoteAddress`](https://github.com/dotnet/dotnet-api-docs/blob/49f51e85e33a589b8aead775f1ac0c47ba2dd27c/xml/System.ServiceModel/ClientBase%601+ChannelBase%601.xml#L718)
- [`.RequestChannel.RemoteAddress`](https://github.com/dotnet/dotnet-api-docs/blob/49f51e85e33a589b8aead775f1ac0c47ba2dd27c/xml/System.ServiceModel/ClientBase%601+ChannelBase%601.xml#L974)

So in theory there shouldn't be any dupes ... Maybe the OPS build needs to be re-queued for this PR, or something?

**Comment by joelmartinez** [comment]
@mairaw actually, my mistake ... my search was filtering out [the duplicate](https://github.com/dotnet/dotnet-api-docs/blob/49f51e85e33a589b8aead775f1ac0c47ba2dd27c/xml/System.ServiceModel/ClientBase%601+ChannelBase%601.xml#L1005). However, it seems this was run without the `--delete` parameter, which is critical for cleaning this particular issue up. I'll work with @dend on re-running this :)

**Pull request merged**